client-js/app/elements/app-link.ts

```diff
@@ -0,0 +1,30 @@
+const MOUSE_MAIN_BUTTON = 0;
+
+@component('app-link')
+@extend('a')
+export class AppLink extends polymer.Base {
+
+  @property({type: Array})
+  path: string[];
+
+  isSpecialClick(e): boolean {
+    return (e.ctrlKey || e.metaKey || e.button !== MOUSE_MAIN_BUTTON);
+  }
+
+  @listen('click')
+  onClick(e) {
+    if (this.isSpecialClick(e)) return;
```

Comment:
LOL; I'm inclined to remove curlies below, but how do you feel about these one-line if statements?